src/download/linux.md

```diff
@@ -42,6 +42,29 @@ apk add prismlauncher
 
 <div class="infobox top">
 
+# <img src="https://cdn.freebiesupply.com/logos/large/2x/altlinux-logo-black-and-white.png" height="20"> ALT Linux / <img src="https://static-00.iconduck.com/assets.00/aosc-icon-512x512-tv7gwhzl.png" height="20"> AOSC
+
+Prism Launcher in both distributions may be behind the current version, so please check the version of Prism Launcher in the repositories before installing. See the dropdown menu for details.
```

Comment:
i'm not really sure if this solves the problem. frankly, if there's a warning about a package (especially when it can be involving known vulnerable versions of prism), i don't think it should be on this page. flatpak would be a much better option in general, and should be the go-to way to use prism when native packages aren't listed here and the user is unsure how to check their repositories for it already being there

Reply:
Okay :)

Reply:
hi sorry for the late review, been pretty busy irl 😆 

i think i was a bit misunderstood here though: given that alt linux is a pretty niche distribution and the official packages don't seem to be maintained well, i don't think this section should be included. after all, i'd imagine users of alt linux who would be safe downloading a possibly older and insecure version of prism could find it in the repositories themselves - plus i think this list should only highlight more notable distributions